.github/actions/windows-patches/action.yaml

```diff
@@ -0,0 +1,111 @@
+name: Run bouf Patch Generation
+description: Generates OBS updater manifest and patches
+inputs:
+  gcsAccessKeyId:
+    description: GCS S3 Access Key ID
+    required: true
+  gcsAccessKeySecret:
+    description: GCS S3 Access Key Secret
+    required: true
+  workflowSecret:
+    description: GitHub API token to use for API calls
+    required: true
+  tagName:
+    description: GitHub Release tag
+    required: true
+  channel:
+    description: Update channel
+    required: false
+    default: 'stable'
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+      with:
+        path: "repo"
+        fetch-depth: 0
+
+    - name: Download Release Artifact
+      shell: pwsh
+      env:
+        GH_TOKEN: ${{ inputs.workflowSecret }}
+      run: |
+        # Download OBS release
+        . ${env:GITHUB_ACTION_PATH}\Invoke-External.ps1
+        Invoke-External gh release download "${{ inputs.tagName }}" -p "*-Windows.zip"
+        Expand-Archive -Path "*-Windows.zip" -DestinationPath "${{ github.workspace }}/build"
+
+    - name: Setup bouf
+      shell: pwsh
+      env:
+        BOUF_TAG: 'v0.6.3'
+        BOUF_HASH: '7f1d266467620aa553a705391ee06128e8ee14af66129a0e64a282997fb6fd83'
+        BOUF_NSIS_HASH: 'a234126de89f122b6a552df3416de3eabcb4195217626c7f4eaec71b20fe36eb'
+        GH_TOKEN: ${{ github.token }}
+      run: |
+        # Download bouf release
+        . ${env:GITHUB_ACTION_PATH}\Ensure-Location.ps1
+        . ${env:GITHUB_ACTION_PATH}\Invoke-External.ps1
+        Ensure-Location bouf
+        $windows_zip = "bouf-windows-${env:BOUF_TAG}.zip"
+        $nsis_zip = "bouf-nsis-${env:BOUF_TAG}.zip"
+        Invoke-External gh release download "${env:BOUF_TAG}" -R "obsproject/bouf" -p $windows_zip -p $nsis_zip
+        
+        if ((Get-FileHash $windows_zip -Algorithm SHA256).Hash -ne "${env:BOUF_HASH}") {
+          throw "bouf hash does not match."
+        }
+        if ((Get-FileHash $nsis_zip -Algorithm SHA256).Hash -ne "${env:BOUF_NSIS_HASH}") {
+          throw "NSIS package hash does not match."
+        }
+        
+        Expand-Archive -Path $windows_zip -DestinationPath bin
+        Expand-Archive -Path $nsis_zip -DestinationPath nsis
+
+    - name: Install rclone and pandoc
+      shell: pwsh
+      run: |
+        choco install rclone pandoc -y --no-progress
```

Comment:
Do we not care about the specific versions of rclone and pandoc here? We pinned to specific versions before. Just curious about the change.

Reply:
I don't really think it's necessary to pin a version, latest should be fine for these. The only reason I pinned it for the signing workflow was the that it's pinned to a specific commit and I wanted it to be consistent until that is updated. But for the publish workflow it doesn't really matter.